spec/tbd_tests_spec.rb

```diff
@@ -989,6 +990,53 @@
     model.save(file, true)
   end
 
+  it "can invalidate KIVA inputs (smalloffice)" do
+    TBD.clean!
+    argh = {}
+    translator = OpenStudio::OSVersion::VersionTranslator.new
+    file = File.join(__dir__, "files/osms/in/smalloffice.osm")
+    path = OpenStudio::Path.new(file)
+    model = translator.loadModel(path)
+    expect(model.empty?).to be(false)
+    model = model.get
+
+    # Reset all ground-facing floor surfaces as "foundations".
+    model.getSurfaces.each do |s|
+      next unless s.outsideBoundaryCondition.downcase == "ground"
+      expect(s.construction.empty?).to be(false)
+      construction = s.construction.get
+      expect(s.setOutsideBoundaryCondition("Foundation")).to be(true)
+      expect(s.setConstruction(construction)).to be(true)
+    end
```

Comment:
Above is an example of a poor SDK-based KIVA setup. It is imprudent to set "Foundation" as surface boundary conditions without prior validation/adaption of KIVA-targeted layered constructions.